python/looker_sdk/sdk/api31/methods.py

```diff
@@ -83,7 +83,6 @@ def login(
             query_params={"client_id": client_id, "client_secret": client_secret},
             transport_options=transport_options,
         )
-        assert isinstance(response, models.AccessToken)
```

Comment:
oops, left over from another branch I'm working on: forgot to `yarn clean && yarn build`